dangerfile.js

```diff
@@ -9,7 +9,7 @@ if (danger.github && danger.github.pr) {
     const { message, url } = commit;
     const [firstLine] = message.split('\n', 1);
 
-    const firstLineRegex = /^(feat|fix|chore|test|docs|refactor|revert)(\(.*\))?:(.+)$/;
+    const firstLineRegex = /^(feat|fix|chore|test|docs|refactor|revert)(\([a-z0-9]*\))?:(.+)$/;
```

Comment:
Consider the following to also prevent empty parens

```suggestion
    const firstLineRegex = /^(feat|fix|chore|test|docs|refactor|revert)(\([a-z0-9]+\))?:(.+)$/;
```